fastlane/Fastfile

```diff
@@ -172,6 +172,14 @@ platform :ios do
       output_files: "riot.unit.test.html,riot.unit.test.report.junit.xml",
       open_report: !is_ci?
     )
+
+    slather(
+      cobertura_xml: true,
+      output_directory: "./build/test",
+      workspace: "Riot.xcworkspace",
+      proj: "Riot.xcodeproj",
+      scheme: "Riot",
```

Comment:
Does not support multiple schemes. We may need to create a new test target running both `Riot` and `RiotSwiftUI` tests, if this is what we want.